### PR TITLE
ebuild-writing/variables: 'live' is also a valid PROPERTIES' value

### PR DESCRIPTION
**Pull request by Flowdalic**
Signed-off-by: Florian Schmaus <flo@geekplace.eu>

### CODE REVIEW

File: ebuild-writing/variables/text.xml [hunk 1]
```diff
@@ -276,7 +276,7 @@ The following variables may or must be defined by every ebuild.
     <ti><c>PROPERTIES</c></ti>
     <ti>
     A space-delimited list of properties, with conditional syntax support.
-    <c>interactive</c> is the only valid value for now.
+    <c>interactive</c> and <c>live</c> are valid values.
```
Comment (ulm) [inline ebuild-writing/variables/text.xml:279]: Maybe  add also `test_network` while at it?
https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-670007.3.5

Comment (Flowdalic) [reply ebuild-writing/variables/text.xml:279]: Sure, but I can't find `test_network` anywhere in the provided link?

Comment (ulm) [reply ebuild-writing/variables/text.xml:279]: https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-670007.3.5

#### 7.3.5 Properties

The following tokens are permitted inside `PROPERTIES`:
<dl>
<dt>interactive</dt><dd>The package may require interaction with the user via the tty.</dd> 
<dt>live</dt><dd>The package uses “live” source code that may vary each time that the package is installed.</dd>
<dt>test_network</dt><dd>The package manager may run tests that require an internet connection, even if the ebuild has <code>RESTRICT=test</code>.</dd>
</dl>
Package managers may recognise other tokens. Ebuilds may not rely upon any token being supported.

**Review by ulm — APPROVED** [review]

**Review by mgorny — APPROVED** [review]
Would be nice to have a more detailed description somewhere but that's irrelated to this change.

**Comment by ulm** [comment]
Thank you.

**Pull request closed without merging**